internal/integrationtest/daemon/daemon_test.go

```diff
@@ -24,14 +24,25 @@ import (
 	"go.bug.st/testsuite"
 )
 
+// FindRepositoryRootPath returns the repository root path
+func FindRepositoryRootPath(t *testing.T) *paths.Path {
+	repoRootPath, err := paths.Getwd()
+	require.NoError(t, err)
+	for !repoRootPath.Join(".git").Exist() {
+		require.Contains(t, repoRootPath.String(), "arduino-cli", "Error searching for repository root path")
+		repoRootPath = repoRootPath.Parent()
+	}
+	return repoRootPath
+}
+
```

Comment:
Why not use `integrationtest.FindRepositoryRootPath` instead of duplicating the function?